core-lib/Kernel.ns

```diff
@@ -424,6 +437,46 @@ class Kernel vmMirror: vmMirror = Object <: Value (
     public print    = ( '#' print. super print )
   )
 
+  public class Character = String ()
+  ( 
+  ) : (
+    public asciiValue: val = (
+
+    )
+    
+    (*convert to char*)
+    public fromValue: val = (
```

Comment:
isn't `from:` sufficient? and the `val` should perhaps be `anInt`? or document the type, please